pac_windows.go

```diff
@@ -87,5 +66,8 @@ func getProxyForURL(pacfileURL, URL string) (string, error) {
 		err = nil
 	}
 
+	defer globalFree.Call(uintptr(unsafe.Pointer(info.lpszProxy)))
+	defer globalFree.Call(uintptr(unsafe.Pointer(info.lpszProxyBypass)))
+	defer globalFree.Call(uintptr(unsafe.Pointer(options.lpszAutoConfigUrl)))
```

Comment:
I'm 99.9% sure that its NOT correct the free this one. Why? Because its different from the other two. the other two represent memory that was allocated by the OS when we called winHttpGetProxyForUrl.  So they don need to be globalFreed.  But options.lpszAutoConfigUrl is an INPUT (not an output). As such, it was allocated by us, and so is known to and will be freed by the Go garbage collector.  So we shouldn't GlobalFree it. (I'm surprised that didn't cause crashes actually).

Reply:
I was actually debating that one but added it to stay on the safe side. I'll remove it.